src/gateways/MsSQL/deleteVisitByCallId.js

```diff
@@ -0,0 +1,39 @@
+import logger from "../../../logger";
+import { statusToId, CANCELLED } from "../../helpers/visitStatus";
+
+const deleteVisitByCallIdGateway = ({ getMsSqlConnPool }) => async (callId) => {
```

Comment:
Perhaps it would be clearer to call this deleteScheduledCallByUuidGateway so that it corresponds to the name of the scheduled_call table?